cost/aws/unused_volumes/README.md

```diff
@@ -1,30 +1,30 @@
-# AWS Unattached Volumes
+# AWS Unused Volumes
 
 ## What it does
 
-This Policy Template scans all volumes in the given account and identifies any unattached volumes that have been unattached for at least the number of user-specified days. If any are found, an incident report will show the volumes, and related information and an email will be sent to the user-specified email address.
+This Policy Template scans all volumes in the given account and identifies any unused volumes and if no read/writes operations performed within the number of user-specified days. If any are found, an incident report will show the volumes, and related information and an email will be sent to the user-specified email address. Cloud watch metrics are used to find the volume read/write operations
 
 If the user approves that the volumes should be deleted, the policy will delete the volumes.
```

Comment:
I know this is left over from the old policy, it doesn't read well.  Please consider :

The volume is deleted after user approval.